Lesson32/Kadir/my-app/src/index.css

```diff
@@ -1,22 +1,22 @@
 * {
   box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
```

Comment:
Text-decoration is applicable to <a> elements and 'list-style' is applicable to list elements. It's very inefficient to apply these styles to all elements in the DOM. Replace with:
```
 * {
 box-sizing: border-box;
 }

a {
text-decoration: none;
color: inherit;
}
ul, ol {
list-style-type: none;
}
```